globus_action_provider_tools/flask/apt_blueprint.py

```diff
@@ -247,6 +243,52 @@ def wrapper() -> ViewReturn:
         self.add_url_rule("/actions", func.__name__, wrapper, methods=["POST"])
         return wrapper
 
+    @overload
+    def action_resume(self, func: Callable[[str, AuthState], ActionStatusReturn]):
+        """
+        Using these stubs w/ @overload tells mypy that the actual implementation
+        for action_status can accept a str or ActionStatus as the first arg type
```

Comment:
Can you update this to `action_resume`?

Reply:
I don't know how I made this mistake after spending so much time thinking carefully exactly what I wanted to write here. :)

Good catch.